### DISCUSSION TITLE
YOLOv8 - Ultralytics YOLOv8 文档

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#7042] [Docs]
# YOLOv8 - Ultralytics YOLOv8 文档

探索YOLOv8的激动人心功能，这是我们实时目标检测器的最新版本！了解高级架构、预训练模型和精确度与速度的最佳平衡如何使YOLOv8成为您进行目标检测任务的理想选择。

https://docs.ultralytics.com/zh/models/yolov8/

**Comment by dusongjie** [comment]
I want add a branch in the backbone network,the point is feeding the input image to two modules,how this way be achieved?

**Reply by pderrenger** [reply]
@dusongjie to add a branch in the backbone network of YOLOv8, you'll need to modify the model's architecture definition file (YAML file). Here's a high-level approach:

1. **Define the New Branch**: In the YAML file, define the new modules you want to add as a branch. This involves specifying the type of layers and connections.

2. **Split the Input**: Adjust the architecture to split the input between the existing backbone and your new branch. You can do this by referencing the input layer in both branches.

3. **Merge Outputs**: If necessary, define how the outputs of the two branches should be combined before proceeding to the next stage of the model.

4. **Update Forward Method**: In the model's Python code, update the `forward` method to handle the new branch. Ensure the input is passed through both branches and the outputs are merged as intended.

5. **Train the Model**: Train the modified model on your dataset to learn the weights for the new branch.

Remember to carefully consider how the new branch will interact with the existing architecture to maintain or improve the model's performance. If you need further guidance, our documentation provides detailed instructions on customizing models. 📘🚀
